pyproject.toml

```diff
@@ -34,10 +34,12 @@ dependencies = [
   "dvc>=3.33.3",
   "dvc-render>=1.0.0,<2",
   "dvc-studio-client>=0.17.1,<1",
+  "dpath",
```

Comment:
is it used only in tests?

Reply:
yeap I changed it to move it into "tests"